MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -197,7 +197,7 @@ public class CarPlayNavigationViewController: UIViewController, MLNMapViewDelega
         let location = notification.userInfo![RouteControllerNotificationUserInfoKey.locationKey] as! CLLocation
         
         // Update the user puck
-        let camera = MLNMapCamera(lookingAtCenter: location.coordinate, fromDistance: 120, pitch: 60, heading: location.course)
+        let camera = MLNMapCamera(lookingAtCenter: location.coordinate, acrossDistance: 120, pitch: 60, heading: location.course)
```

Comment:
While I don't mind this change, it is a bit weird, that this is here. Is this intentional? Why is this changed? This seems to be unrelated to SPM?

Reply:
This is the warning I got and thats why I changed it:

```
'init(lookingAtCenter:fromDistance:pitch:heading:)' is deprecated: Use -cameraLookingAtCenterCoordinate:acrossDistance:pitch:heading: or -cameraLookingAtCenterCoordinate:altitude:pitch:heading:.
```

![Bildschirmfoto 2024-04-12 um 13 31 01](https://github.com/maplibre/maplibre-navigation-ios/assets/18739004/6a130395-b19f-45ce-a983-fc4e7d3ff49c)